### PR TITLE
feat(plugin-wdm): add autologout policy support

### LINKED ISSUE #134: Migrate policy

### PR DESCRIPTION
**Pull request by ianwremmel**
Fixes #134

### CODE REVIEW

File: packages/plugin-wdm/src/device/device.js [hunk 1]
```diff
@@ -194,9 +206,49 @@ const Device = SparkPlugin.extend({
...
     this.set(res.body);
+  },
+
+  _resetLogoutTimer() {
+    clearTimeout(this.logoutTimer);
+    this.unset(`logoutTimer`);
+    if (this.intranetInactivityCheckUrl && this.intranetInactivityDuration) {
+      this.on(`change:lastUserActivityDate`, () => this._resetLogoutTimer());
+
+      const timer = setTimeout(() => {
```
Comment (mwegman) [inline packages/plugin-wdm/src/device/device.js]: So you're going to set the timeout no matter what? Even if you're on the intranet?

File: packages/plugin-conversation/src/conversation.js [hunk 2]
```diff
@@ -504,6 +504,11 @@ const Conversation = SparkPlugin.extend({
       });
     }
 
+    // leaky abstraction
+    if (activity.verby !== `acknowledge`) {
```
Comment (mwegman) [inline packages/plugin-conversation/src/conversation.js]: verby?

File: packages/plugin-wdm/src/device/device.js (hunk 1 above)
Comment (ianwremmel) [inline packages/plugin-wdm/src/device/device.js]: Isn't that a requirement? There's no guarantee they'll be on the intranet eight hours from now, right?

Comment (mwegman) [inline packages/plugin-wdm/src/device/device.js]: I figured if the user switched networks, we would no about it. I don't know how that would happen without a 'network down' and a 'network back up'

Comment (mwegman) [inline packages/plugin-wdm/src/device/device.js]: If the user can move between networks and we don't know about it, your way is the way to go.

**Comment by Josh-Dykstra** [comment]
LGTM

**Pull request merged**